spec/routing/journals_routing_spec.rb

```diff
@@ -4,31 +4,31 @@
   describe "routing" do
 
     it "routes to #index" do
-      get("/journals").should route_to("journals#index")
+      expect(get: "/journals").to route_to("journals#index")
     end
 
     it "routes to #new" do
-      get("/journals/new").should route_to("journals#new")
+      expect(get: "/journals/new").to route_to("journals#new")
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.